package.json

```diff
@@ -1,12 +1,20 @@
 {
   "name": "@azure/ng-deploy",
-  "version": "0.1.0-beta.3",
+  "version": "0.1.0-beta.4",
+  "main": "out/ng-add/index.js",
+  "files": [
+    "out/",
+    "builders.json",
+    "collection.json"
+  ],
   "description": "@azure/ng-deploy - Deploy Angular apps to Azure using the Angular CLI",
   "scripts": {
-    "build": "tsc -p tsconfig.json",
+    "build": "tsc -p tsconfig.json && npm run copy:builders:json && npm run copy:ngadd:json",
```

Comment:
`npm start` doesn't work as expected anymore. It should keep running in watch mode. Instead, the process finishes.